pkgs/development/python-modules/amazon-ion/default.nix

```
@@ -1,54 +1,69 @@
 { lib
 , buildPythonPackage
+, docopt
 , fetchFromGitHub
 , jsonconversion
-, six
 , pytestCheckHook
 , pythonOlder
+, setuptools
+, six
+, tabulate
 }:
 
 buildPythonPackage rec {
   pname = "amazon-ion";
-  version = "0.10.0";
-  format = "setuptools";
+  version = "0.11.2";
+  pyproject = true;
 
   disabled = pythonOlder "3.7";
 
-  # test vectors require git submodule
   src = fetchFromGitHub {
     owner = "amzn";
```

```suggestion
    owner = "amazon-ion";
```

Repository has been moved to https://github.com/amazon-ion/ion-python